distributor/Dockerfile

```diff
@@ -19,4 +19,4 @@ COPY  --from=build /usr/src/target /usr/src/target
 WORKDIR /usr/src/
 ADD ./web web
 
-CMD java -jar target/bin/distributor.jar --eclk.distributor.db.url=jdbc:mysql://$DB_URL --eclk.distributor.db.username=$DB_UN --eclk.distributor.db.password=$DB_PW --eclk.distributor.db.useSsl=$DB_USE_SSL --eclk.govsms.username=$SMS_USER  --eclk.govsms.password=$SMS_PASSWORD --eclk.govsms.source=$SMS_SOURCE
+CMD java -jar target/bin/distributor.jar --eclk.distributor.db.url=jdbc:mysql://$DB_URL --eclk.distributor.db.username=$DB_UN --eclk.distributor.db.password=$DB_PW --eclk.distributor.db.useSsl=$DB_USE_SSL --eclk.distributor.w1=$W1 --eclk.distributor.w2=$W2 --eclk.distributor.w3=$W3 --eclk.distributor.w4=$W4 --eclk.distributor.w5=$W5 --eclk.govsms.username=$SMS_USER  --eclk.govsms.password=$SMS_PASSWORD --eclk.govsms.source=$SMS_SOURCE
```

Comment:
Let's use a single variable for worker URL

Reply:
Updated.